docs/Dockerfile

```diff
@@ -6,31 +6,17 @@ ENV DEBIAN_FRONTEND=noninteractive \
 
 # General requirements not in the base image
 RUN apt-get -qqy update \
-  && apt-get -qqy install \
-    ca-certificates \
-    curl \
-    git \
-    npm \
-    python3 \
-    python3-distutils \
-    python3-pip \
-    python3.9 \
-    python3-distutils \
-    python3.9-venv \
-    software-properties-common \
-    tzdata \
-    sudo \
-    unzip \
-  # Set Python 3.9 as the default
-  && update-alternatives --install /usr/bin/python3 python3 /usr/bin/python3.8 1 \
-  && update-alternatives --install /usr/bin/python3 python3 /usr/bin/python3.9 2;
-
-# Make sure we're using the latest pip
-RUN pip install --upgrade pip \
-  && pip install virtualenv
+  && apt-get -qqy install git npm python3.9-venv
 
 WORKDIR /app/
 
-COPY ./package.json ./
+COPY package.json requirements.txt .
```

Comment:
At least locally this fails for me without writing the directory as `./`.

Reply:
I guess we have different version of Docker, both work for me. I've put back `./`.